meson.build

```diff
@@ -0,0 +1,182 @@
+project(
+  'cloud-init',
+  meson_version: '>=0.63.0', # rockylinux/9
+  license: 'GPL-3 or Apache-2.0',
+  default_options: [
+    # The default can yield broken results.
+    'python.install_env=auto',
+  ],
+)
+system = host_machine.system()
+sysconfdir = get_option('sysconfdir')
+INIT_SYSTEM = get_option('init_system')
+
+lib_exec_dir = join_paths(get_option('prefix'), get_option('libexecdir'), 'cloud-init')
+
+pymod = import('python')
+python = pymod.find_installation('python3')
+
+#https://mesonbuild.com/FAQ.html#but-i-really-want-to-use-wildcards for downsteam drop-in files
+find = find_program('find')
+
+install_subdir(
+  'cloudinit',
+  install_dir: python.get_install_dir(),
+  install_tag: 'python',
+)
+
+completions = dependency('bash-completion')
+# Binaries and script entrypoints
+bash_completions_dir = completions.get_variable(
+  pkgconfig: 'completionsdir',
+  default_value: join_paths(get_option('datadir'), 'bash-completion', 'completions'),
+)
+completions = dependency('bash-completion')
+install_data(
+  'bash_completion/cloud-init',
+  install_dir: bash_completions_dir,
+  install_tag: 'scripts',
+)
+
+install_data(
+  [
+    'tools/ds-identify',
+    'tools/hook-hotplug',
+    'tools/uncloud-init',
+    'tools/write-ssh-key-fingerprints',
+  ],
+  install_dir: lib_exec_dir,
+  install_tag: 'scripts',
+)
+
+install_data(
+  ['tools/cloud-init-per', 'scripts/cloud-id', 'scripts/cloud-init'],
+  install_mode: 'rwxr-xr-x',
+  install_dir: get_option('bindir'),
+)
+
+# Required Config and Templates
+render_tmpl = './tools/render-template'
+
```

Comment:
The location of this variable assignment is odd. Generally variables should be set immediately before use, or in the case of constant / static data like this, near the top.

Reply:
Will pull it to the top, since it's used in 3 separate places.